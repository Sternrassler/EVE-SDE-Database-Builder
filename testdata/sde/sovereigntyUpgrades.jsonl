{"upgradeID":1,"upgradeName":"Minmatar Infrastructure Hub Upgrade","description":"Infrastructure upgrade for sovereignty"}
{"upgradeID":2,"upgradeName":"Caldari Infrastructure Hub Upgrade","description":"Infrastructure upgrade for sovereignty"}
{"upgradeID":3,"upgradeName":"Amarr Infrastructure Hub Upgrade","description":"Infrastructure upgrade for sovereignty"}
