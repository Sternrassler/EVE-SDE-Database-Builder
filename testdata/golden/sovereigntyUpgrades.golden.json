[
  {
    "upgradeID": 1,
    "typeID": null,
    "level": null
  },
  {
    "upgradeID": 2,
    "typeID": null,
    "level": null
  },
  {
    "upgradeID": 3,
    "typeID": null,
    "level": null
  }
]